lib/workers/repository/update/branch/get-updated.spec.ts

```diff
@@ -500,5 +500,21 @@ describe('workers/repository/update/branch/get-updated', () => {
         ],
       });
     });
+
+    it('handles replacement', async () => {
+      config.upgrades.push({
+        packageFile: 'index.html',
+        manager: 'html',
+        updateType: 'replacement',
+        branchName: undefined!,
+      });
+      autoReplace.doAutoReplace.mockResolvedValueOnce('my-new-dep:1.0.0');
+      const res = await getUpdatedPackageFiles(config);
+      expect(res).toMatchSnapshot({
```

Comment:
```suggestion
      expect(res).toMatchObject({
```

Please read our best practices guide in `docs/development` directory

Reply:
done